src/main/java/seedu/address/logic/commands/BorrowCommand.java

```diff
@@ -0,0 +1,97 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_BOOKS;
+import static seedu.address.model.book.BookStatus.STATUS_DATE_FORMAT;
+
+import java.text.ParseException;
+import java.util.Date;
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.book.Book;
+import seedu.address.model.patron.Patron;
+
+/**
+ * Parses input arguments and creates a new BorrowCommand object
+ */
+public class BorrowCommand extends Command {
+
+    public static final String MESSAGE_USAGE = BORROW_COMMAND_WORD
+            + ": Establishes a borrow relationship between the patron identified by the index number "
+            + " in the displayed patron list and the book identified the index number in the displayed book list.\n"
+            + "Parameters: PATRON_INDEX BOOK_INDEX (both must be positive integers) "
+            + "RETURN_DATE (in dd-MMM-yyyy format and after today's date)\n"
+            + "Example: " + BORROW_COMMAND_WORD + " 1 1 28-Feb-2023";
```

Comment:
Is there a typo in the example? ```" 1 1 28-Feb-2023";```

Reply:
No, there is no typo. It is intentionally set to 2023 because the return date needs to always be after today's date.